friends/src/App.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react';
+import './App.css';
+import axios from 'axios';
+import Form from './Components/Form/Form';
+class App extends Component {
+  constructor() {
+    super();
+    this.state={
+      friends:[],
+      friend: ''
+    };
+    const inputHandler = e => {
+      this.setState({ friend: e.target.value });
+    };
+    const addFriend = e => {
```

Comment:
`addFriend` should be making a POST request with axios. I can see what you're going for though! Here's what I would put in here:
```
axios
  .post('http://localhost:5000/friends', this.state.friend)
  .then(response => {
    this.setState({
      friends: response.data,
      friend: { name: '', age: '', email: '' },
    });
  })
  .catch(err => console.error(err));
```

This is assuming that `this.state.friend` is an object.